lib/Runtime/Library/WebAssemblyInstance.cpp

```diff
@@ -300,7 +300,14 @@ void WebAssemblyInstance::LoadImports(
                 AsmJsScriptFunction* func = AsmJsScriptFunction::FromVar(prop);
                 if (!wasmModule->GetWasmFunctionInfo(counter)->GetSignature()->IsEquivalent(func->GetSignature()))
                 {
-                    JavascriptError::ThrowWebAssemblyLinkError(ctx, WASMERR_SignatureMismatch);
+                    char16 temp[1024] = { 0 };
```

Comment:
> Add a test case exercising this code?

We do test this code, which is why the PR is currently failing CI - I need to update the baselines.

> The signatures can be very long, are we sure the buffers are big enough ?

I can change to heap allocation and growing the buffer when needed, but that complicates the lifetime tracking slightly.